docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,151 @@
+init(const char *name)
+{
+   Evas_Object *main_win;
+
+   /* create main window for the application */
+   main_win = elm_win_util_standard_add(name, name);
+
+   /* set up main window */
+   ...
+   evas_object_show(main_win);
+
+
+   tzsh_h tzsh;
+   tzsh_quickpanel_h tzsh_qp;
+   tzsh_window tz_win;
+
+   /* Get native window ID of main window */
+   tz_win = elm_win_window_id_get(main_win);
+
+   /* Create tzsh_h structure */
+   tzsh = tzsh_create(TZSH_TOOLKIT_TYPE_EFL);
+
+   /* Create tzsh_quickpanel_h structure */
+   tzsh_qp = tzsh_quickpanel_create_with_type(tzsh, tz_win, TZSH_QUICKPANEL_TYPE_SYSTEM_DEFAULT);
+```
+
+
+## Showing and Hiding Quickpanel Window
+
+To show and hide quickpanel window when your application's window is activated, call the following functions:
+
+```
+   ...
+
+   if (show)
+     tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
+   else
+     tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
+
+   ...
+```
+
+## Getting Visible Status of Quickpanel Window
+To know the state of current visibility of qp service's window, call the `tzsh_quickpanel_visible_get` function. The quickpanel service's window may be visible or not visible on the screen.
+
+```
+   ...
+   tzsh_quickpanel_state_visible_e state;
+
+   tzsh_quickpanel_visible_get(tzsh_qp, &state);
+   if (state == TZSH_QUICKPANEL_STATE_VISIBLE_SHOWN)
+     /* visible state */
+     ;
+   else if (state == TZSH_QUICKPANEL_STATE_VISIBLE_HIDDEN)
+     /* invisible state */
+     ;
+   else
+     /* error */
+     ;
+```
+
+
+## Registering Changed Event for the Quickpanel Window
+To be notified of state changes, implement the appropriate event callback function and call the `tzsh_quickpanel_event_handler_add` function with that. If you want to change your application's behavior to match visibility of the quicknapel service's window, then you should handle state change event as following:
+
+```
+static tzsh_quickpanel_event_handler_h handler;
+
+static void
+ev_callback(int type, tzsh_quickpanel_event_info_h ev_info, void *data)
+{
+   tzsh_quickpanel_state_visible_e state;
+
+   if (type != TZSH_QUICKPANEL_EVENT_VISIBLE)
+     return;
+
+   tzsh_quickpanel_event_visible_get(ev_info, &state);
+   if (state == TZSH_QUICKPANEL_STATE_VISIBLE_SHOWN)
+     ;
+   else if (state == TZSH_QUICKPANEL_STATE_VISIBLE_HIDDEN)
+     ;
+   else
+     ;
+}
+
+static void
+init_tzsh(Evas_Object *main_win)
+{
+   ...
+
+   /* register event callback */
+   handler = tzsh_quickpanel_event_handler_add(tzsh_qp, TZSH_QUICKPANEL_EVENT_VISIBLE, ev_callback, NULL);
+
+   ...
+}
+```
+
+
+## Destroying TZSH Quickpanel and TZSH Structures
+When no longer needed, free structures with following functions:
+
+```
+static void
+deinit(void)
+{
+   tzsh_quickpanel_event_handler_del(tzsh_qp, handler);
+   tzsh_quickpanel_destroy(tzsh_qp);
+   tzsh_destroy(tzsh);
+}
+```
+
+
+## Related Information
+- Dependencies
+  - Tizen 3.0 and Higher for Mobile and Wearable
```

Comment:
Please separate this info as like other pages to get the consistency

  - Tizen 3.0 and Higher for Mobile
  - Tizen 3.0 and Higher for Wearable

Reply:
Fine. I will change it as your comments.